cli/edbook/conf/settings.py

```
@@ -518,8 +518,25 @@ def get_authors(config: EdbookConfig):
+            setup_path = Path(__file__).parent.parent / "setup.py"
+            with open(setup_path, 'r') as f:
+                content = f.read()
+                match = re.search(r'version="([^"]*)"', content)
+                if match:
+                    return match.group(1)
+        except Exception as e:
+            print(f"Warning: Could not read version from setup.py: {e}")
+        return year  # fallback to year if version can't be read
+    
+    edbook_version = get_edbook_version()
+    version = edbook_version  # The short X.Y version.
```

Ruff (0.12.2)

Local variable `version` is assigned to but never used

Remove assignment to unused variable `version`

(F841)